gatsby-config.js

```diff
@@ -55,6 +55,12 @@ module.exports = {
         doc_lang: "en",
       },
     },
+    {
+      resolve: `gatsby-source-github-contributors`,
```

Comment:
We should probably use an API token for GitHub as in Buzz About to prevent throttling issues.